WebApp/src/main/java/ev/projects/webapp/controllers/CaseController.java

```diff
@@ -0,0 +1,47 @@
+package ev.projects.webapp.controllers;
+
+import ev.projects.models.Case;
+import ev.projects.services.ICaseService;
+import ev.projects.webapp.forms.SearchFormData;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+@RequestMapping("/cases")
+public class CaseController {
+
+    private ICaseService caseService;
+
+    @Autowired
+    public CaseController(ICaseService caseService) {
+        this.caseService = caseService;
+    }
+
+    @PostMapping("/get-case")
+    public String redirectToCasePage(@ModelAttribute SearchFormData searchFormData) {
+        return "redirect:/cases/" + searchFormData.getID();
+    }
+
+    @GetMapping("/{id}")
+    public String showCase(@PathVariable("id") long ID, Model model) {
```

Comment:
Why not just name the method `getCase`?

Reply:
Thought that because this is MVC maybe a better word would be show, but get looks also good. Will rename it.

Reply:
Renamed in 203629b8e9c2c87e4ee27a87b1abc11cc3fc6f6c.